src/Dumplie/Infrastructure/Doctrine/Dbal/Metadata/Field/TextMapping.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Infrastructure\Doctrine\Dbal\Metadata\Field;
+
+use Doctrine\DBAL\Schema\Table;
+use Dumplie\Application\Metadata\Schema\FieldDefinition;
+use Dumplie\Application\Metadata\Schema\Type;
+use Dumplie\Infrastructure\Doctrine\Dbal\Metadata\TypeMapping;
+
+class TextMapping implements TypeMapping
+{
+    /**
+     * @var Type
+     */
+    private $type;
+
+    /**
+     * TextMapping constructor.
+     */
+    public function __construct()
+    {
+        $this->type = new Type('text');
+    }
+
+    /**
+     * @return Type
+     */
+    public function type(): Type
+    {
+        return $this->type;
+    }
+
+    /**
+     * @param Table           $table
+     * @param string          $name
+     * @param FieldDefinition $definition
+     *
+     * @return bool
+     */
+    public function map(Table $table, string $name, FieldDefinition $definition): bool
```

Comment:
this method seems to be a command and a query at one time. It would be good to extract checking if this specific TypeMapping is allowed for FieldDefinition::type(). Maybe you could compare Type VOs?